addons/account/models/account_move_line.py

```diff
@@ -62,6 +62,11 @@ class AccountMoveLine(models.Model):
         copy=False,
         group_operator='min',
     )
+    invoice_date = fields.Date(
+        related='move_id.invoice_date', store=True,
+        copy=False,
```

Comment:
> Why is this needed ?

The "Journal Items" view is based on the "account.move.line" model so I want that field to be in the model so I can display it.



> You also should remove https://github.com/odoo/enterprise/blob/61d81a378482e790c78f5084c89d8271f3e35858/account_followup/models/account_move_line.py#L16 since you added the property in the base model.

Will be done ! :ok_hand:

Reply:
@jbw-odoo I want the behavior on invoice_date to be the same as on the date. If the accounting date ("date" field) should not be copied, I guess the invoice date should not be copied either. Should it ?